docs/building.md

```diff
@@ -1,3 +1,5 @@
+_Note: this doc is out of date_
+
```

Comment:
But the gsutil info is still useful, and may not be recorded elsewhere yet.

Reply:
We should probably just update to reference `bin/plugin`.